src/Reducers/index.js

```diff
@@ -1,5 +1,5 @@
 import { combineReducers } from 'redux';
-import searchResultReducer from './searchResultReducer';
+import searchResultReducer from '../State/SearchResult/reducer';
```

Comment:
i think for the file names we had was reducers plural just like in the other reducers file names

Reply:
added plural